plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -799,6 +799,37 @@ SecRule REQUEST_FILENAME "@rx /remote\.php/dav/addressbooks/users/(?:[^/]+/){2}[
         "t:none,\
         ctl:ruleRemoveById=950100"
 
+# Deleting a contact from addressbook
+# XML parser fails because a content type is set without a request body
+SecRule REQUEST_FILENAME "@rx /remote\.php/dav/addressbooks/users/[^/]+/[^/]+/[^/]+\.vcf$" \
+    "id:9508324,\
+    phase:2,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.2.0',\
+    chain"
+    SecRule REQUEST_METHOD "@streq DELETE" \
+        "t:none,\
+        chain"
+        SecRule REQUEST_BODY_LENGTH "@eq 0" \
+            "t:none,\
+            ctl:ruleRemoveById=200002"
+
+# Syncing contacts via a dav client
+SecRule REQUEST_FILENAME "@rx /remote\.php/dav/addressbooks/users/[^/]+/(?:[^/]+/)?$" \
+    "id:9508325,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.2.0',\
+    ctl:ruleRemoveTargetById=942430;XML:/*,\
+    ctl:ruleRemoveTargetById=942431;XML:/*,\
+    ctl:ruleRemoveTargetById=942432;XML:/*,\
+    ctl:ruleRemoveTargetById=942440;XML:/*,\
+    ctl:ruleRemoveTargetById=921110;REQUEST_BODY"
```

Comment:
I'd move this last line to the start of the list, just because of alphabetical ordering.